examples/client/dub.json

```diff
@@ -0,0 +1,10 @@
+{
+	"name": "client",
+	"homepage": "https://github.com/trishume/ddbus",
+	"copyright": "Copyright © 2017, Tristan Hume",
+	"authors": ["Tristan Hume"],
+	"targetType": "executable",
+	"dependencies": {
```

Comment:
I think limiting this file to `{"name":"client"}` is enough for an example dub.json because people should be able to easily copy the example

Reply:
Hmm, at least it seems that `"dependeices" { "ddbus": ..}` is needed for building.

Reply:
oh yeah sorry, name and dependencies